pkg/disruption/ci/remote_sampler.go

```diff
@@ -1,112 +0,0 @@
-package ci
-
-import (
-	"context"
-	"sync"
-
-	"github.com/openshift/origin/pkg/disruption/backend/sampler"
-	"github.com/openshift/origin/pkg/monitor/backenddisruption"
-	"github.com/openshift/origin/pkg/monitor/monitorapi"
-	"k8s.io/kubernetes/test/e2e/framework"
-
-	"k8s.io/client-go/rest"
-	"k8s.io/client-go/tools/events"
-)
-
-type testRemoteFactory struct {
-	dependency dependency
-	err        error
-}
-
-// RemoteSampler has the machinery to start disruption monitor in the cluster
-type RemoteSampler struct {
```

Comment:
what replaced this?

Reply:
We no longer need a dedicated sampler - in-cluster daemonsets will use the new-style disruption samplers